docs/sql/postgresql/migration_1.5.3_1.6.0.sql

```diff
@@ -0,0 +1,25 @@
+-- Changeset powerauth-java-server/1.6.x/20231018-add-constraint-operation-template-name.xml::1::Jan Pesek
+-- Add unique constraint to pa_operation_template.template_name
+ALTER TABLE pa_operation_template ADD CONSTRAINT pa_operation_template_template_name_uk UNIQUE (template_name);
+
+-- Changeset powerauth-java-server/1.6.x/20231103-add-activation-name-history.xml::1::Lubos Racansky
+-- Add activation_name column to pa_activation_history
+ALTER TABLE pa_activation_history ADD activation_name VARCHAR(255);
+
+-- Changeset powerauth-java-server/1.6.x/20231106-add-foreign-keys.xml::1::Jan Pesek
+ALTER TABLE pa_operation_application ADD CONSTRAINT pa_operation_application_application_id_fk FOREIGN KEY (application_id) REFERENCES pa_application (id);
+
+-- Changeset powerauth-java-server/1.6.x/20231106-add-foreign-keys.xml::2::Jan Pesek
+ALTER TABLE pa_operation_application ADD CONSTRAINT pa_operation_application_operation_id_fk FOREIGN KEY (operation_id) REFERENCES pa_operation (id);
+
+-- Changeset powerauth-java-server/1.6.x/20231112-add-activation-id.xml::1::Jan Dusil
+-- Add activation_id column to pa_operation with foreign key constraint
+ALTER TABLE pa_operation ADD activation_id VARCHAR(37);
+
+ALTER TABLE pa_operation ADD CONSTRAINT pa_operation_activation_id_fk FOREIGN KEY (activation_id) REFERENCES pa_activation (activation_id);
+
+-- Changeset powerauth-java-server/1.6.x/20231123-operation-user-nullable.xml::1::Roman Strobl
+-- Make user_id column in table pa_operation nullable
+ALTER TABLE pa_operation ALTER COLUMN  user_id DROP NOT NULL;
+
+-- Changeset powerauth-java-server/1.6.x/20231212-add-tag-1.6.0.xml::1::Lubos Racansky
```

Comment:
A comment without actual statement? Shouldn't here be some `DATABASECHANGELOG` insert?

Reply:
This generated script does not insert anything into `databasechangelog`, even the tag.